configs/alcf/config.20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta

```diff
@@ -33,15 +33,15 @@ ncclimoParallelMode = bck
 ## options related to reading in the results to be analyzed
 
 # directory containing model results
-baseDirectory = /projects/OceanClimate/mpeterse/20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta/run
+baseDirectory = /lus/theta-fs0/projects/OceanClimate_2/mpeterse/20171031.tenYearTest.GMPAS-IAF.T62_oEC60to30v3wLI.60layer.theta/run
 
 # names of ocean and sea ice meshes (e.g. oEC60to30v3, oQU240v3, oRRS30to10v3, etc.)
 mpasMeshName = oEC60to30v3wLI
 
 # Directory for mapping files (if they have been generated already). If mapping
 # files needed by the analysis are not found here, they will be generated and
 # placed in the output mappingSubdirectory
-mappingDirectory = /projects/ccsm/acme/diagnostics_data/mapping
+mappingDirectory = /lus/theta-fs0/projects/ccsm/acme/diagnostics_data/mapping
```

Comment:
@xylar: I think we should change the location of these directories to $projdir/diagnostics/mpas_analysis/maps in all the example config files.

Reply:
@milenaveneziani, thanks for catching this.  I think there are a couple of things still to address.  First, the public observations are, as you know, incomplete so we would need to move the other data sets (WOCE, Schmidtko, etc.) to the same location.  Second, there are people with only access to `OceanClimate_2` and others with only access to `ClimateEnergy_2`, so we can't safely put shared files in either of those projects.  This is why we probably want to put observations, mapping files, etc. in the `ccsm/acme` folder.  There is a similar need on OLCF.  However, this means they are *not* in `$project`, which needs to be a scratch or project directory for a particular allocation if I understand right.